igvc_perception/src/cnn/cnn.py

```diff
@@ -1,42 +1,40 @@
-#!/usr/bin/env python
 import cv2
 import functools
-import message_filters
 import numpy as np
-import pdb
 from PIL import Image
 import sys
-from timeit import default_timer as timer
 import torch
 from torch.autograd import Variable
 from torchvision import transforms
 
 import os
-sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/../")
 from train_eval.models.model import UNet
 
 # ROS imports.
 import rospy
 import tf as ros_tf
 
-from cv_bridge import CvBridge, CvBridgeError
+from cv_bridge import CvBridge
 from image_geometry import PinholeCameraModel
 from sensor_msgs.msg import CameraInfo
 from sensor_msgs.msg import CompressedImage
 from sensor_msgs.msg import Image as ImMsg
 from sensor_msgs.msg import PointCloud2
 from sensor_msgs.msg import PointField
 
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/../")
+
```

Comment:
We actually can't have this below the `from train_eval.models.model import UNet`, as if you look at the directory structure, `train_eval` is like located in the parent directory, if you know what I mean.

Anyways, I think the proper way of fixing this is to remove this line, and then move the `train_eval` directory inside the `cnn` directory, so we can do `from train_eval.models.model import UNet` normally without any jank path manipulations

Reply:
Also, it turns out that the rest of our files are have a bunch of PEP8 violations... 😅